samples/AnomalyDetector/src/Generated/Models/FillNAMethod.cs

```diff
@@ -53,7 +53,7 @@ public FillNAMethod(string value)
 
         /// <inheritdoc />
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public override int GetHashCode() => _value?.GetHashCode() ?? 0;
+        public override int GetHashCode() => StringComparer.InvariantCultureIgnoreCase.GetHashCode(_value);
```

Comment:
Shall we still do the 'null' check for _value?  
As this is a struct, people could create a new instance without calling the constructor.

Reply:
this method handles null values properly.

Reply:
well I thought this API handles null value properly but it turns out that it does not, it just simply throw when there is a null value.
I will update the code and add a test case